src/esp/nav/PathFinder.cpp

```diff
@@ -134,6 +145,9 @@ class IslandSystem {
         if (navMesh->isValidPolyRef(startRef) &&
             (polyToIsland_.find(startRef) == polyToIsland_.end())) {
           uint32_t newIslandId = islandRadius_.size();
+          islandsToPolys_.emplace(std::piecewise_construct,
+                                  std::forward_as_tuple(newIslandId),
+                                  std::tuple<>());
```

Comment:
```suggestion
```
Looking closer at the code, I don't think we actually need this part at all. the '[]' operator will default initialize a value if it doesn't exist and luckily the std::vector is default constructible. So as long you don't access it with a lower level call like .at(), this is unnecessary.